src/components/TopicMenu/TopicAccordions/TopicAccordion.tsx

```diff
@@ -15,7 +14,7 @@ class Topic {
 
 function TopicAccordion(props: any) {
     const routing = useNavigate()
-
+    const topicsWithoutArrow = ['Magic Items', 'Monsters']
```

Comment:
I'm not sure on the context of this, but is there a way to infer this list based on something from the api? If so I like to avoid hard coding lists like this.

Reply:
I don't think so, but im willing to be wrong. The purpose of this list is to let the accordion know whether to have a show more icon.